src/postfork.cpp

```diff
@@ -415,10 +456,31 @@ void safe_report_exec_error(int err, const char *actual_cmd, const char *const *
             FLOGF_SAFE(exec, "Out of memory");
             break;
         }
-
+        case EACCES: {
+            FLOGF_SAFE(exec, "Failed to execute process '%s': The file could not be accessed.", actual_cmd);
+            break;
+        }
+        case ETXTBSY: {
+            FLOGF_SAFE(exec, "Failed to execute process '%s': File is currently open for writing.", actual_cmd);
```

Comment:
I specifically made these messages *not* just the ones we get by default.

That's because "text file is busy" is among the worst error messages I have ever read, mostly because it has a 98% chance of appearing when the file is *not* a text file (in the sense any user cares about).

It's kinda like if an ice cream place told you "machine not fillable" when they're out of a certain flavor. Might be true in the technical sense, but so misleading that it might as well not be said, at all.

Reply:
So I would welcome more wording suggestions.